data/frame.go

```diff
@@ -44,30 +45,24 @@ type Frame struct {
 
 // UnmarshalJSON uses the `UnmarshalArrowFrame` function to unmarshal this type from JSON.
 func (f *Frame) UnmarshalJSON(b []byte) error {
-	arrow := []byte{}
-
-	if err := json.Unmarshal(b, &arrow); err != nil {
-		return err
-	}
-
-	frame, err := UnmarshalArrowFrame(arrow)
-	if err != nil {
-		return err
-	}
-
-	*f = *frame
-
-	return nil
+	return readDataFrameJSON(f, b)
 }
 
 // MarshalJSON uses the `MarshalArrow` function to marshal this type to JSON.
 func (f *Frame) MarshalJSON() ([]byte, error) {
-	arrow, err := f.MarshalArrow()
+	cfg := jsoniter.ConfigCompatibleWithStandardLibrary
```

Comment:
@kminehart -- is this going to be OK?  it used to write a base64 string, but IIRC you don't really need that anywhere

Reply:
Correct, I thought I would need to marshal / unmarshal a frame with caching but it turns out I didn't need it. :)  you're good

Reply:
One benefit though, I did add a test that made sure that the data that was marshaled from a frame unmarshaled into a matching frame. :)

Reply:
I have the same test -- the only hitch is that the dates only have millisecond precision, so I needed to add:
https://github.com/grafana/grafana-plugin-sdk-go/pull/293/files#diff-0a0e41e03a8757f5e4b6adf13759a5a464e909334ccfaf8602832f22efc1aeb3R386

I think that is ok, and we should look at using a floating point value for time when > 1ms resolutoin is required